crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -115,8 +115,8 @@ def start_step(step_name)
           raise Crowbar::Error::StartStepExistenceError.new(step_name)
         end
         if running?
-          @logger.warn("Some step is already running.")
-          raise Crowbar::Error::StartStepRunningError.new
+          @logger.warn("Step #{current_step} is already running.")
+          raise Crowbar::Error::StartStepRunningError.new(step_name = current_step)
```

Comment:
@Itxaka you're right, I though you were passing `step_name` there not `current_step`.